mixins/global.js

```diff
@@ -1,11 +1,6 @@
 // Global mixin so that every component can access the logged in state and user.
 import Vue from 'vue'
-import {
-  REGION_LONDON,
-  REGION_WALES,
-  TAGLINE_LONDON,
-  TAGLINE_WALES,
-} from '@/regions'
+import { REGION_WALES, TAGLINE_LONDON, TAGLINE_WALES } from '@/regions'
```

Comment:
Is REGION_LONDON no longer needed?

Reply:
Checking the code made me realise that that showSubmitBusiness() shouldn't be used any more as we want to show the submit link for REW.

That does leave REGION_LONDON not explicitly used much.  But that's because it's the default; e.g. see businessesInBounds() in BusinessPage.vue, where we use REGION_WALES and default to London.  So we still need to keep it.